src/app/App.jsx

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 import Root from "../routes/Root";
+import ReactModal from "react-modal";
 import "./App.scss";
 import "./components/hero/Hero.scss";
+import "./components/pages/Pages.scss";
+import "./components/pages/FavouriteList.scss";
+import "./components/pages/MovieDetails.scss";
+import "./components/pages/Hamburger.scss";
+import "./components/pages/RandomMovies.scss";
```

Comment:
Zrób to w ten sposób.

<img width="743" alt="image" src="https://github.com/Szersz87/just_movies/assets/20505844/6d4cc66b-68a0-4ab0-a21b-3d946ab6cbfa">

Reply:
zrobione